utils/toggle-negation.ts

```
@@ -75,8 +74,8 @@ function toggleBinaryExpression(
   node: BinaryExpression,
   context: Rule.RuleContext,
 ): string {
-  let left = getNodeContent(node.left, context).trim()
-  let right = getNodeContent(node.right, context).trim()
+  let left = context.sourceCode.getText(node.left).trim()
```

ESLint

[error] Unsafe argument of type error typed assigned to a parameter of type `Node | undefined`.

(typescript/no-unsafe-argument)